src/types/contracts/derivativeScrt/model.ts

```diff
@@ -0,0 +1,20 @@
+import { StakingDerivativesValidator } from './response';
+
+type StakingDerivativesInfo = {
```

Comment:
derivativeScrtInfo  to be consistent unless there's another name you like more. Would prefer to be more specific so that when it's imported into a project, the type is intuitive that we are talking specifically about stkd-SCRT.